diff --git a/node_modules/@github/copilot-sdk/dist/cjs/client.js b/node_modules/@github/copilot-sdk/dist/cjs/client.js
index 1dc0634..d8fac71 100644
--- a/node_modules/@github/copilot-sdk/dist/cjs/client.js
+++ b/node_modules/@github/copilot-sdk/dist/cjs/client.js
@@ -639,6 +639,7 @@ class CopilotClient {
         streaming: config.streaming,
         includeSubAgentStreamingEvents: config.includeSubAgentStreamingEvents ?? true,
         mcpServers: toWireMcpServers(config.mcpServers),
+        githubMcpToolOptions: config.githubMcpToolOptions,
         envValueMode: "direct",
         customAgents: toWireCustomAgents(config.customAgents),
         defaultAgent: config.defaultAgent,
@@ -766,6 +767,7 @@ class CopilotClient {
         streaming: config.streaming,
         includeSubAgentStreamingEvents: config.includeSubAgentStreamingEvents ?? true,
         mcpServers: toWireMcpServers(config.mcpServers),
+        githubMcpToolOptions: config.githubMcpToolOptions,
         envValueMode: "direct",
         customAgents: toWireCustomAgents(config.customAgents),
         defaultAgent: config.defaultAgent,
diff --git a/node_modules/@github/copilot-sdk/dist/client.js b/node_modules/@github/copilot-sdk/dist/client.js
index 1f3c400..283d5fa 100644
--- a/node_modules/@github/copilot-sdk/dist/client.js
+++ b/node_modules/@github/copilot-sdk/dist/client.js
@@ -625,6 +625,7 @@ class CopilotClient {
         streaming: config.streaming,
         includeSubAgentStreamingEvents: config.includeSubAgentStreamingEvents ?? true,
         mcpServers: toWireMcpServers(config.mcpServers),
+        githubMcpToolOptions: config.githubMcpToolOptions,
         envValueMode: "direct",
         customAgents: toWireCustomAgents(config.customAgents),
         defaultAgent: config.defaultAgent,
@@ -752,6 +753,7 @@ class CopilotClient {
         streaming: config.streaming,
         includeSubAgentStreamingEvents: config.includeSubAgentStreamingEvents ?? true,
         mcpServers: toWireMcpServers(config.mcpServers),
+        githubMcpToolOptions: config.githubMcpToolOptions,
         envValueMode: "direct",
         customAgents: toWireCustomAgents(config.customAgents),
         defaultAgent: config.defaultAgent,
